app/design/adminhtml/default/default/template/eav/attribute/options.phtml

```diff
@@ -166,6 +186,27 @@ var attributeOption = {
             this.updateItemsCountField();
         }
     },
+    swatch : function(event){
+        var element = $(Event.findElement(event, 'tr')); // !!! Button already
```

Comment:
I'd totally avoid using prototypejs

Reply:
I didn't remove prototypejs to avoid having mixed code, I think it can be the subject of a future PR

Reply:
I think we shouldn't add prototypejs based new code, it would be less work/testing later when we'll finally convert everything to vanillajs.

anyway, I'll not oppose to this but also I can't collaborate on this PR because it's locked

Reply:
what do you say we do this in a next PR? I don't have time now